src/common/wallet-state.ts

```diff
@@ -303,7 +304,7 @@ function useWalletState(initialState?: Partial<WalletStateParams>): WalletData {
   const syncStatus = getOrElse(
     (): SynchronizationStatus => ({
       mode: 'offline',
-      currentBlock: -1,
+      currentBlock: 0,
```

Comment:
Wallet doesn't load if the fetched sync status is offline with currentBlock == 0.
It was -1 to make sure it gets updated by every possible fetched sync status.
Feel free to change the logic in `refreshSyncStatus` if you don't like having -1 here.

Reply:
Fixed